python/tests/test_pyvw.py

```diff
@@ -210,3 +210,33 @@ def test_example_namespace():
     assert exm.pop_feature()
     exm.push_features(nmid, ['c', 'd'])
     assert exm.num_features_in() == 4
+
+def test_simple_label():
+    vw_ex = vw(quiet=True)
+    ex = vw_ex.example('1 |a two features |b more features here')
+    sl = pyvw.simple_label('c', weight=0.5)
+    assert sl.label == 'c'
+    assert sl.weight == 0.5
+    assert sl.prediction == 0.0
+    assert sl.initial == 0.0
+    assert str(sl) == 'c:0.5'
+    sl2 = pyvw.simple_label(ex)
```

Comment:
You are testing whether this is equal against itself? It will always be true.

Reply:
Can you please suggest some other way to test them? Or should I test it numerically as in `sl`

Reply:
Do as you did in the previous 3 lines, where you assert against known values